README.md

```diff
@@ -30,6 +30,36 @@ export function point(lat, long) {
 
 See all examples in the [examples folder](/examples). Use `yarn examples` to apply any changes to all the examples.
 
+## Trigger functions
+
+To write a trigger function, decorate the function with the `//@plv8ify-trigger` comment, and have the function return a `testRow` type where `testRow` defines the type of the row for the trigger. You can also add a NEW parameter for insert and update triggers, and OLD for update and delete triggers.
+(Tip: you can add @types/pg and @types/plv8-internals to get all standard postgres types/defines and plv8 specific functions recognized by the type checker)
+
+```
+type testRow = {
```

Comment:
Is it a convention to call it `testRow`? How about just `Row` or `TestRow` (matching casing for types) or `RowType`?

Reply:
> Is it a convention to call it `testRow`?

no, not at all, naming of the type is independent as it will only be used for type-checking in typescript. WHat matters is that the fields in the type match the table record names and types so that the trigger does not error out when run in plv8.
will change it to Row